File: pfm-sdk/src/main/java/com/tink/pfmsdk/util/ColorsUtils.java
package com.tink.pfmsdk.util;


import androidx.core.graphics.ColorUtils;

public class ColorsUtils {

	public static int adjustAlpha(int baseColor, float factor) {
		int alpha = Math.round(255 * factor);
		return ColorUtils.setAlphaComponent(baseColor, alpha);
	}

}

File: moneymanager-ui/src/main/java/com/tink/moneymanagerui/util/ColorsUtils.kt
package com.tink.moneymanagerui.util

import androidx.core.graphics.ColorUtils
import kotlin.math.roundToInt

object ColorsUtils {
    fun adjustAlpha(baseColor: Int, factor: Float): Int {
        val alpha = (255 * factor).roundToInt()
        return ColorUtils.setAlphaComponent(baseColor, alpha)
    }
}